### PR TITLE
test: fix opentelemetry test

### PR DESCRIPTION
**Pull request by povilasv**
# Required for all PRs

- [ ] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

resolves #

Fixes opentelemetry test mock server returning:
```
 rpc error: code = Internal desc = grpc: error while marshaling: proto: Marshal called
```
 
 See the deleted comment for more details

### CODE REVIEW

File: plugins/outputs/opentelemetry/opentelemetry_test.go [hunk 1]
```diff
@@ -66,13 +66,7 @@ func TestOpenTelemetry(t *testing.T) {
...
 
 	err = plugin.Write([]telegraf.Metric{input})
-	if err != nil {
-		// TODO not sure why the service returns this error, but the data arrives as required by the test
-		// rpc error: code = Internal desc = grpc: error while marshaling: proto: Marshal called with nil
-		if !strings.Contains(err.Error(), "proto: Marshal called with nil") {
-			assert.NoError(t, err)
-		}
-	}
+	assert.NoError(t, err)
```
Comment (srebhan) [inline plugins/outputs/opentelemetry/opentelemetry_test.go:69]: Can you please use `require.NoError` as we do prefer this over `assert` if possible.

**Review by srebhan — COMMENTED** [review]
Thanks @povilasv for your fix! One small comment from my side...

**Review by srebhan — APPROVED** [review]
Looks good to me. Thanks for cleaning this up @povilasv!

**Review by powersj — APPROVED** [review]
Thanks for the fix!

**Pull request merged**